core/domain/skill_jobs_one_off.py

```diff
@@ -116,3 +116,63 @@ def reduce(key, values):
                 sum(ast.literal_eval(v) for v in values))])
         else:
             yield (key, values)
+
+
+class SkillCommitCmdMigrationOneOffJob(jobs.BaseMapReduceOneOffJobManager):
```

Comment:
Do you know why this issue happened in the first place? I think, for all these job PRs, it's worth also including a brief explanation for (a) why it happened, (b) how to prevent it from happening again.

Reply:
Same as above.